Codescene.VSExtension.VS2022/Codescene.VSExtension.Core/Application/Services/CodeReviewer/CodeReviewer.cs

```diff
@@ -94,93 +94,66 @@ public DeltaResponseModel Delta(FileReviewModel review, string currentCode)
-
-            if (refactoredFunctions == null)
+            if (!refactorableFunctions.Any())
             {
-                throw new Exception("Refactoring has failed!");
+                _logger.Debug("No refactorable functions found. Skipping update of delta cache.");
+                return true;
             }
-
-            var cacheItem = new CachedRefactoringActionModel
-            {
-                Path = path,
-                RefactorableCandidate = f,
-                Refactored = refactoredFunctions
-            };
-
-            //_cache.Add(cacheItem); Use new cache impl, but also, should we cache this? ACE already has cache on the API side. We don't cache it in JB.
-
-            return cacheItem;
+            return false;
         }
 
-        public async Task<RefactorResponseModel> Refactor(string path, FnToRefactorModel refactorableFunction, bool invalidateCache = false)
+        private void UpdateFindings(DeltaResponseModel delta, IList<FnToRefactorModel> refactorableFunctions)
         {
-            if (string.IsNullOrWhiteSpace(refactorableFunction.FunctionType))
+            foreach (var finding in delta.FunctionLevelFindings)
             {
-                refactorableFunction.FunctionType = "MemberFn";
-            }
-
-            var refactorableFunctionsString = JsonConvert.SerializeObject(refactorableFunction);
+                var functionName = finding.Function?.Name;
+                if (string.IsNullOrEmpty(functionName))
+                    continue;
 
-            var refactoredFunctions = await _executer.PostRefactoring(fnToRefactor: refactorableFunctionsString, skipCache: true);
-
-            if (refactoredFunctions == null)
-            {
-                throw new Exception("Refactoring has failed!");
+                UpdateFindingIfNotUpdated(finding, functionName, refactorableFunctions);
             }
+        }
 
-            var cacheItem = new CachedRefactoringActionModel
+        private void UpdateFindingIfNotUpdated(FunctionFindingModel finding, string functionName, IList<FnToRefactorModel> refactorableFunctions)
+        {
+            // update only if not already updated, for case when multiple methods have same name
+            if (finding.RefactorableFn == null)
             {
-                Path = path,
-                RefactorableCandidate = refactorableFunction,
-                Refactored = refactoredFunctions
-            };
-
-            //_cache.Add(cacheItem);
-
-            return refactoredFunctions;
+                var match = refactorableFunctions.FirstOrDefault(fn => fn.Name == functionName && checkRange(finding, fn));
+                if (match != null)
+                {
+                    finding.RefactorableFn = match;
+                }
+            }
         }
 
-        public CachedRefactoringActionModel GetCachedRefactoredCode()
+        private bool checkRange(FunctionFindingModel finding, FnToRefactorModel refFunction)
```

Comment:
Respecting naming convention for methods
```suggestion
        private bool CheckRange(FunctionFindingModel finding, FnToRefactorModel refFunction)
```